src/Domain/Package/PackageManager.php

```diff
@@ -130,6 +130,38 @@ public function getAll(): array {
     return $this->packages;
   }
 
+  /**
+   * Gets an array of all company packages.
+   *
+   * @return \Acquia\Orca\Domain\Package\Package[]
+   *   An array of packages keyed by package name.
+   */
+  public function getCompanyPackages(): array {
+    $companyPackages = [];
+    foreach ($this->packages as $package) {
+      if ($package->isCompanyPackage()) {
+        $companyPackages[$package->getPackageName()] = $package;
+      }
+    }
+    return $companyPackages;
+  }
+
+  /**
+   * Gets an array of all dependencies.
+   *
+   * @return \Acquia\Orca\Domain\Package\Package[]
+   *   An array of packages keyed by package name.
+   */
+  public function getDependencies(): array {
```

Comment:
I wonder if we should make the distinction here a little more explicit, since company packages are technically dependencies, too. Like this, maybe?

```suggestion
  public function getThirdPartyDependencies(): array {
```

Or this?

```suggestion
  public function getVendorDependencies(): array {
```

Reply:
I will go with `getThirdPartyDependencies `